test/e2e/support/utils.js

```diff
@@ -61,6 +61,11 @@ export async function visitAdmin( adminPath, query ) {
 
 export async function newPost( postType ) {
 	await visitAdmin( 'post-new.php', postType ? 'post_type=' + postType : '' );
+
+	// Disable new user tips so that their UI doesn't get in the way
```

Comment:
A NUX seems like a good candidate for E2E testing, since a developer is unlikely to encounter it in their day-to-day work, and thus is more prone to accidental breakage.

Reply:
Definitely. I'll follow up on this later though since the PR is already quite large. Issue to track: https://github.com/WordPress/gutenberg/issues/7037